src/main/java/com/yugabyte/sample/apps/MultiTableMultiIndexInserts.java

```diff
@@ -0,0 +1,449 @@
+        }
+    }
+    LOG.info("Created all the indexes");
+  }
+
+  public long doRead() {
+    return 1L;
+  }
+
+  private UUID getNextUUID(int threadIdx) {
+    PerThreadState threadState = threadStates.get(threadIdx);
+    long a = threadState.getNextLong();
+    long b = threadState.getNextLong();
+    return new UUID(a, b);
+  }
+
+  private int getNextInt(int threadIdx) {
+    PerThreadState threadState = threadStates.get(threadIdx);
+    return threadState.getNextInt();
+  }
+
+  private PreparedStatement getPreparedStatement(int threadIdx, String baseTableName) {
+    PerThreadState threadState = threadStates.get(threadIdx);
+    return threadState.getPreparedStatement(getTableName(baseTableName));
+  }
+
+  private Connection getConnection(int threadIdx) {
+    PerThreadState threadState = threadStates.get(threadIdx);
+    return threadState.getConnection();
+  }
+
+  private static final String ALPHA_NUMERIC_STRING = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+  String getRandomString(int threadIdx) {
+    Random random = threadStates.get(threadIdx).getRandom();
+    StringBuilder builder = new StringBuilder();
+    int count = 7;
+    while (count-- != 0) {
+      int character = random.nextInt(ALPHA_NUMERIC_STRING.length());
+      builder.append(ALPHA_NUMERIC_STRING.charAt(character));
+    }
+    return builder.toString();
+  }
+
+  private PreparedStatement getA1Statement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = getPreparedStatement(threadIdx, "a1");
+    int count = rowsCount.get("a1");
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setObject(2, getNextUUID(threadIdx));
+      preparedStatement.setObject(3, getNextUUID(threadIdx));
+      preparedStatement.setInt(4, getNextInt(threadIdx));
+      preparedStatement.setInt(5, getNextInt(threadIdx));
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getA1AStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = getPreparedStatement(threadIdx, "a1a");
+    int count = rowsCount.get("a1a");
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setObject(2, getNextUUID(threadIdx));
+      preparedStatement.setInt(3, getNextInt(threadIdx));
+      preparedStatement.setInt(4, getNextInt(threadIdx));
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getA2Statement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = getPreparedStatement(threadIdx, "a2");
+    Connection conn = getConnection(threadIdx);
+    int count = rowsCount.get("a2");
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setString(2, "\"name:de=Nowosibirsk\"");
```

Comment:
Maybe replace this with a more generic sample value?

Reply:
Done.